charts/nsq/values.yaml

```diff
@@ -134,10 +135,190 @@ nsqd:
 
   service:
     type: ClusterIP
+    annotations:
 
   persistence:
     enabled: true
     size: 1Gi
     storageClass: "-"
     accessModes:
       - ReadWriteOnce
+
+## Prometheus Exporters / Metrics
+##
+metrics:
+  enabled: false
+  image:
+    registry: docker.io
+    repository: lovoo/nsq_exporter
+    tag: 2.0.2
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
```

Comment:
It's odd to have this many comments on typical config keys, here at the bottom of values.yaml but not near the top for the first component/image of the chart. I suggest removing the stock boilerplate comments for config keys that exist in 95% of charts, like `pullPolicy`. That will make this diff much shorter and easier to review.

Reply:
ok, agree more